datasets/youtube.py

```diff
@@ -3,30 +3,40 @@
 import argparse
 import glob
 
+def clean_up_transcript_ref(ref):
```

Comment:
why is this needed? isn't this handled by preprocessor? are there other offending punct marks?

Reply:
We use the replica separator as a semicolon.
If a semicolon is encountered in the reference transcription, the asert in encode_transcript func stops working.
assert len(ref_by_speaker) == len(t['speaker'])

Reply:
please put semicolon in a properly named variable and add a comment

Reply:
done